pkg/inspector/image-inspector.go

```diff
@@ -336,6 +336,7 @@ func appendDockerCfgConfigs(dockercfg string, cfgs *docker.AuthConfigurations) e
 	if err != nil {
 		return fmt.Errorf("Unable to open docker config file: %v\n", err)
```

Comment:
@enoodle if the error is not nil then the reader will be nil so it's ok to skip this return